runtest.py

```diff
@@ -298,7 +311,7 @@ def main():
     arguments = docopt(__doc__)
     run_test_obj = RunTest(exec_config=arguments['--exec-config'], firefox_config=arguments['--firefox-config'],
                            index_config=arguments['--index-config'], online_config=arguments['--online-config'],
-                           global_config=arguments['--global-config'])
+                           global_config=arguments['--global-config'], chrome_config=arguments['--chrome-config'])
```

Comment:
No, we don't need to add `CHROME_CONFIG` due to `FIREFOX_CONFIG` only be used by following method to check if there has video recording and fps validation. These parts should be extracted out from browser related config.